src/panels/lovelace/views/hui-view-header.ts

```diff
@@ -51,6 +52,10 @@ export class HuiViewHeader extends LitElement {
     this._checkHidden();
   };
 
+  private _dragScrollController = new DragScrollController(this, {
```

Comment:
The click action should be prevented when you have dragged the badges, not sure how to achieve that here...